setup.cfg

```diff
@@ -24,6 +24,7 @@ include_package_data = true
 python_requires = >=3.7
 install_requires =
 	pydantic >= 1.9.1
+	typing_extensions >= 3.7.2
```

Comment:
Why 3.7.2? I looked up the changelog, but it directs to the git history for versions prior to 4.0. I don't want to add a specific version unless there was a specific reason for the minimum pertinent to this project.

Reply:
I believe that Literal wasn't added until this version. Any version can be picked as long as it's after this one.

Reply:
Okay. I did confirm the introduction of `Literal` appears in the [changelog between 3.6.6 and 3.7.4](https://github.com/python/typing_extensions/compare/3.6.6...3.7.4). There's no tag for 3.7.2, unfortunately, leaving it difficult to confirm that the feature was added in that version. Since we're not certain and because those releases are already years old and because this project is likely to depend on features later than that release, I'll just leave it unspecified (when I merge).